packages/chart/src/components/BarChart/components/BarChart.Horizontal.tsx

```diff
@@ -68,7 +68,16 @@ export const BarChartHorizontal = () => {
               <Group className={`bar-group-${barGroup.index}-${barGroup.x0}--${index} ${config.orientation}`} key={`bar-group-${barGroup.index}-${barGroup.x0}--${index}`} id={`bar-group-${barGroup.index}-${barGroup.x0}--${index}`} top={barGroup.y}>
                 {barGroup.bars.map((bar, index) => {
                   const scaleVal = config.useLogScale ? 0.1 : 0
-                  const { suppresedBarHeight: suppresedBarWidth, getIconSize, getIconPadding, getVerticalAnchor, isSuppressed } = composeSuppressionBars({ bar })
+                  const { suppresedBarHeight: suppresedBarWidth, getIconSize, getVerticalAnchor, isSuppressed } = composeSuppressionBars({ bar })
+
+                  const conditions = {
```

Comment:
Nit: It might be better to use functions outside of an object in the future. If these are helpers you could consider moving them to their own file. I'm not sure that the word conditions conveys a lot here. You might want to fix the spelling error in `shoMissingDataLabel`.

Example:
`/helpers/index.ts`

` const getBarWidth = () => `

Reply:
Fixed